Transformers/Transformers.ipynb

```diff
@@ -0,0 +1,278 @@
+{
```

Comment:
<p>Yay! The train, evaluate is cleaner than earlier. But also, opaque on what they're doing internally in terms of datasets and writing to disk (e.g. model files) specially. </p><p> </p><p>Should we parameterize these functions to make it explicit instead?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NirantK/Hinglish/pull/33/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
<blockquote>Should we parameterize these functions to make it explicit instead?</blockquote><p> </p><p>I didn't completely understand this part, how do we go about doing this? Is there any example I can refer to?</p>